app/client/src/components/shared/MapWidgets.tsx

```diff
@@ -1794,6 +1793,7 @@ function ShowSurroundingMonitoringLocations({
         });
       });
   }, [
+    abortController,
```

Comment:
Refs don't trigger re-renders, so this is an extra dependency.

Reply:
Good point.